__tests__/libs/ontuml2gufo/relations.test.ts

```diff
@@ -5,47 +5,65 @@ describe('Relations', () => {
   let alpinebitsResult;
   let derivationResult;
   let partWholeResult;
+  let partWholeHideRelationResult;
 
   beforeAll(async () => {
```

Comment:
There are not tests using multiple parameters. 
In particular, we should test how `hideObjectPropertyCreation` and `createInverses` behave together, as well as `createInverses` and `prefixPackages`.